sdk/python/kfp/utils.py

```diff
@@ -0,0 +1,61 @@
+def use_secret(secret_name:str=None, volume_name:str=None, secret_volume_mount_path:str=None, env_variable:str=None, secret_file_path_in_volume:str=None):
```

Comment:
Do you mind also add the license statement on top of the two files? Thanks

Reply:
Where can i find examples of the "license statement"?

Reply:
Oh it's the following comment block which you can find in almost every python file in this repo.

```python
# Copyright 2019 Google LLC
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#      http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```